CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Main (unreleased)
 ### Breaking changes
 
 - The experimental dynamic configuration feature has been removed in favor of Flow mode. (@mattdurham)
+- The `oracledb` integration configuration has removed a redundant field `metrics_scrape_interval`, `scrape_interval` usage of the integration is recommended. (@schmikei)
```

Comment:
I'm not sure I understand what's being recommended here.

This states that the integration configuration has changed, and that using the integration is recommended.

I understand the removal of the redundant field, but..

Has the use of the integration not always been the best-practice recommendation? What has changed here?

Reply:
Ahh, okay, the change of the parameters makes much more sense now. I don't think you need to elaborate on the change in the changelog.

I think my confusion comes from the grammar. Maybe a change like this would be more clear, assuming I'm understanding it correctly.

```suggestion
- The `oracledb` integration configuration has removed a redundant field `metrics_scrape_interval`. Use the `scrape_interval` parameter of the integration if a custom scrape interval is required. (@schmikei)
```